src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1529,6 +1530,24 @@ public function getContent($asResource = false)
         return $this->content;
     }
 
+    /**
+     * Return request body content as a json decoded array.
+     *
+     * @return mixed
+     *
+     * @throws JsonException
+     */
+    public function getJsonDecoded()
+    {
+        $json = json_decode($this->getContent(), true, 512, JSON_BIGINT_AS_STRING);
```

Comment:
I'm uncertain, if I would expect the `JSON_BIGINT_AS_STRING` to be set when calling that method. Maybe we should allow the flags to be passed as argument to the `getJsonDecoded` method?